lib/dependencies/HarmonyCompatibilityDependency.js

```diff
@@ -70,15 +70,16 @@ HarmonyCompatibilityDependency.Template = class HarmonyExportDependencyTemplate
 		}
 		if (moduleGraph.isAsync(module)) {
 			runtimeRequirements.add(RuntimeGlobals.module);
-			const used = exportsInfo.isUsed(runtime);
-			if (used) runtimeRequirements.add(RuntimeGlobals.exports);
+			runtimeRequirements.add(RuntimeGlobals.asyncModule);
 			initFragments.push(
 				new InitFragment(
-					`${module.moduleArgument}.exports = (async () => {\n`,
+					`${RuntimeGlobals.asyncModule}(${module.moduleArgument}, async (__webpack_handle_async_dependencies__) => {\n`,
```

Comment:
What about IE 11 and other old browsers, `async`/`await` are not available, we already have issues about these problems

Reply:
This doesn't make it worse. `async` was used before.

This is very difficult to fix, since we would need to run a regenerator-like step here to. See the other issue, where I explained this in detail.

Reply:
What is our strategy for these issues in future (workaround)?

Reply:
we met this problems, maybe webpack should add warnings if user want to use TLA on legacy target?